### PR TITLE
Feature/show nest quick control

### PR DESCRIPTION
**Pull request by larsblumberg**
Make nest thermostat's quick control (value up/down) appear and let the user change the target temperature

@DaQuirm @ghophp

### CODE REVIEW

File: public/app/common/components/controls/slider/slider.jsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ var SliderControl = React.createClass({
 
   onChange: function onChange (e) {
     this.setState({ value: e.target.value }, () => {
-      this.props.channel(this.state);
```
Comment (DaQuirm) [inline public/app/common/components/controls/slider/slider.jsx]: OK, channel was more FRPish

File: public/app/common/components/device-action-control/device-action-control.jsx [hunk 2]
```diff
@@ -6,28 +6,30 @@ var DeviceHelpers = require('helpers/device');
...
   getControlProps: function getControlState (device) {
     var capability = device.primaryCapability;
     var property = DeviceHelpers.propertyByCapability[capability];
     var value = device.alterEgoComponent.properties[property];
-    return { value: value, channel: this.generateAction };
+    return { value: value, onChange: this.onChangeInvokeAction };
   },
 
-  generateAction: function generateActionData (state) {
+  onChangeInvokeAction: function onChangeInvokeAction (state) {
```
Comment (DaQuirm) [inline public/app/common/components/device-action-control/device-action-control.jsx]: The new name is not very meaningful to me, that is, the `onChange` prefix is vague, it doesn't specify on _which_ change. In any case I'm against emphasising that a method acts an event handler in general. "Event handler" names only tell _when_ a method is called but not what it actually does. :pouting_cat:

File: public/app/common/helpers/device/device-helpers.js [hunk 3]
```diff
@@ -24,15 +24,8 @@ var Helpers = {
   },
 
   getAlterEgoComponent: function getAlterEgoComponent (device) {
-    // TODO: remove this ugly hack when mainComponentId is there
-    if (device.properties.displayType === 'LAMP') {
-      return _.find(device.components, (component) =>
-        component.properties.type === 'SOCKET'
-      );
-    }
-
```
Comment (DaQuirm) [inline public/app/common/helpers/device/device-helpers.js]: Oh thank God :smiley_cat:

File: public/app/common/components/device-action-control/device-action-control.jsx (hunk 2 above)
Comment (larsblumberg) [inline public/app/common/components/device-action-control/device-action-control.jsx]: I agree with you here. Is `invokeAction` a good handler name?

However, if the method handles the event itself, I still prefer naming methods with the `on<Event>` pattern.

Comment (DaQuirm) [inline public/app/common/components/device-action-control/device-action-control.jsx]: I vote for `invokeValueChangeAction`

**Pull request merged**